doc/README_Template.md

```diff
@@ -0,0 +1,119 @@
+
+YaST &lt;FOO&gt; Module

```

Comment:
Later, it would also make sense to link a good README file that shows how we expect the docs to look like. Example is IMO needed.